<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EELEditor</class>
 <widget class="QDialog" name="EELEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EEL Script Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QMenuBar" name="menuBar">
     <property name="defaultUp">
      <bool>false</bool>
     </property>
     <widget class="QMenu" name="menuFile">
      <property name="title">
       <string>File</string>
      </property>
      <addaction name="actionOpen_file"/>
      <addaction name="actionSave"/>
      <addaction name="actionClose_filw"/>
     </widget>
     <widget class="QMenu" name="menuEdit">
      <property name="title">
       <string>Edit</string>
      </property>
      <addaction name="separator"/>
      <addaction name="actionUndo"/>
      <addaction name="actionRedo"/>
      <addaction name="separator"/>
      <addaction name="actionFind_Replace"/>
      <addaction name="actionGo_to_line"/>
     </widget>
     <widget class="QMenu" name="menuNavigation">
      <property name="title">
       <string>Navigation</string>
      </property>
      <addaction name="separator"/>
      <addaction name="separator"/>
      <addaction name="actionJump_to_function"/>
      <addaction name="separator"/>
      <addaction name="actionGo_to_init"/>
      <addaction name="actionGo_to_sample"/>
     </widget>
     <addaction name="menuFile"/>
     <addaction name="menuEdit"/>
     <addaction name="menuNavigation"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSplitter" name="splitter_main">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_sidebar">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="ProjectView" name="projectView">
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
        </widget>
        <widget class="CodeOutline" name="outline">
         <property name="indentation">
          <number>20</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Outline</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CodeEditor" name="codeEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="FindReplaceForm" name="findReplaceForm" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_filw">
   <property name="text">
    <string>Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="text">
    <string>Find/Replace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionGo_to_line">
   <property name="text">
    <string>Go to Line...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionJump_to">
   <property name="text">
    <string>Jump to...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionJump_to_function">
   <property name="text">
    <string>Jump to function...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionPrevious_function">
   <property name="text">
    <string>Previous function...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionGo_to_init">
   <property name="text">
    <string>Go to @init</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionGo_to_sample">
   <property name="text">
    <string>Go to @sample</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAutomatic_Parentheses">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic Parentheses</string>
   </property>
  </action>
  <action name="actionReplace_Tabs_with_Spaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Replace Tabs with Spaces</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FindReplaceForm</class>
   <extends>QWidget</extends>
   <header>widgets/findreplaceform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CodeOutline</class>
   <extends>QTreeWidget</extends>
   <header>widgets/codeoutline.h</header>
  </customwidget>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QTextEdit</extends>
   <header>widgets/codeeditor.h</header>
  </customwidget>
  <customwidget>
   <class>ProjectView</class>
   <extends>QListWidget</extends>
   <header>widgets/projectview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
